### ISSUE TITLE
[EN][quest][The Shattered Halls] A very special cloth

### ISSUE THREAD

**Issue by arvit** [Atlantiss/BugTracker#9510]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

The boss [Warbringer O'mrogg](http://www.wowhead.com/npc=16809/warbringer-omrogg)  doesn't drop the [item](http://www.wowhead.com/item=72597/omroggs-warcloth) needed for the quest or the drop rate is too low.
Tested on heroic version.

**Steps to reproduce the problem**:

1. Get quest
2. Kill boss
3. No quest item

**Revision**: 5488

http://www.wowhead.com/npc=16809/warbringer-omrogg
http://www.wowhead.com/item=72597/omroggs-warcloth
http://www.wowhead.com/quest=29655/a-very-special-cloth

**Comment by tomatosz** [comment]
If I recall correctly, this quest (like many) only works on normal on atlantiss.

**Comment by arvit** [comment]
Yeah it got reported before and fixed https://github.com/Atlantiss/BugTracker/issues/797 for normal mode.

**Comment by wtRCH** [comment]
Should be rewarded only on Normal Difficulty as far as I know
